### PR TITLE
add initial support for events

### PR DESCRIPTION
**Pull request by dejanb**

### CODE REVIEW

File: broker/etc/broker.xml [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+            <address-full-policy>PAGE</address-full-policy>
+            <auto-create-queues>true</auto-create-queues>
+            <auto-create-addresses>true</auto-create-addresses>
+            <auto-create-jms-queues>true</auto-create-jms-queues>
+            <auto-create-jms-topics>true</auto-create-jms-topics>
+         </address-setting>
+      </address-settings>
+
+      <addresses>
+         <address name="event/DEFAULT_TENANT">
```
Comment (sophokles73) [inline broker/etc/broker.xml]: does this mean that we need to create addresses for tenants and their mapping to a queue upfront, or can these queues be *auto-created* based on a pattern, e.g. `event/${TENANT}` maps to queue `event.${TENANT}`?

Comment (dejanb) [inline broker/etc/broker.xml]: Broker can be configured to auto-create/delete addresses and queues as needed. This was just a first POC configuration. It's on the improvement list.

Comment (sophokles73) [inline broker/etc/broker.xml]: I see. FMPOV we should have this *improvement* implemented for the next milestone, though. Would you agree?

Comment (dejanb) [inline broker/etc/broker.xml]: Definitely. I'm looking now into reenabling Artemis check in IT and docs. Next on the list is setting ssl and this address configuration. And that should be good for the next milestone. Do you see anything else?

Comment (sophokles73) [inline broker/etc/broker.xml]: no, sounds great :-)

**Pull request merged**